.travis.yml

```diff
@@ -3,8 +3,6 @@ node_js:
   - "4"
   - "5"
   - "6"
-script:
```

Comment:
That's Travis default for Node projects I believe.
On 17 May 2016 1:17 AM, "Hugo Giraudel" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/kaelig/sass-project-boilerplate/pull/1#discussion_r63370416>
> :
>
> > @@ -3,8 +3,6 @@ node_js:
> >    - "4"
> >    - "5"
> >    - "6"
> > -script:
>
> Shouldn’t we have something like:
>
> script:
>   - npm test
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaelig/sass-project-boilerplate/pull/1/files/4c6935f6f309b5617eadc1d7c7f49d23f64a9ede#r63370416>
>